spec/models/article_destroy_spec.rb

```diff
@@ -22,8 +22,11 @@
     it "queues BustCacheJob with user and organization article_ids" do
       expect do
         article.destroy
-      end.to have_enqueued_job(Articles::BustMultipleCachesJob).exactly(:once).
-        with([user_article.id, org_user_article.id, org_article.id].sort)
+      end.to change(Articles::BustMultipleCachesWorker.jobs, :size).by(1)
+
+      enqueued_job = Articles::BustMultipleCachesWorker.jobs.pop
+
+      expect(enqueued_job["args"].first).to eq([user_article.id, org_user_article.id, org_article.id].sort)
```

Comment:
You can replace this with `sidekiq_assert_enqueued_with` if you want, to be a little bit more elegant. 

This is an example:

https://github.com/thepracticaldev/dev.to/blob/863904ac0628e16a7e6f301f452c8a7e654805f2/spec/workers/rate_limit_checker_worker_spec.rb#L6-L8

Reply:
awesome!